proto/cosmos/bank/v1beta1/tx.proto

```diff
@@ -34,13 +34,13 @@ message MsgSend {
 // MsgSendResponse defines the Msg/Send response type.
 message MsgSendResponse {}
 
-// MsgMultiSend represents an arbitrary multi-in, multi-out send message.
+// MsgMultiSend represents a single input, multi-out send message.
 message MsgMultiSend {
-  option (cosmos.msg.v1.signer) = "inputs";
+  option (cosmos.msg.v1.signer) = "input";
 
   option (gogoproto.equal) = false;
 
-  repeated Input  inputs  = 1 [(gogoproto.nullable) = false];
+  Input  input  = 1 [(gogoproto.nullable) = false];
```

Comment:
This PR got merged with a proto breaking change, and the proto package did not get bumped.

We should actually revert this change. @likhita-809 could you create a follow-up PR?

Reply:
(thinking about it, my comment https://github.com/cosmos/cosmos-sdk/pull/12610#discussion_r923294345 might have been confusing... I thought there was something to change in the signer proto option, but there's actually nothing to change).

Reply:
So, should I revert it to this commit https://github.com/cosmos/cosmos-sdk/pull/12610/commits/abc4340f878dd03a91aa14c93d1122cdd76f965a#diff-4b06aeb8c7fd43a99b1a1c0eb62792aae06649ead661bd26bc843e9a0485336a

Reply:
Not 100% sure which commit it should be, but all the proto changes in this PR should be reverted.